src/commands/protocols/update/sender/update-validate-asset-command.js

```diff
@@ -48,26 +47,13 @@ class UpdateValidateAssetCommand extends ValidateAssetCommand {
             operationId,
             blockchain,
         );
-        const isValidAssertion = await this.validationService.validateDatasetRoot(
-            cachedData.dataset,
-            datasetRoot,
-        );
+        this.validationService.validateDatasetRoot(cachedData.dataset, datasetRoot);
```

Comment:
```suggestion
       await this.validationService.validateDatasetRoot(cachedData.dataset, datasetRoot);
```

await is missing, right?

Reply:
Do you do any on-chain validation here, not sure why it would as this is pre mint validation

Reply:
There shouldnt be any onchains call for this

Reply:
There should be an await my bad